cluster/gce/gci/configure-helper-common.sh

```diff
@@ -3197,12 +3176,10 @@ function ensure-master-bootstrap-kubectl-auth {
   # If the insecure port is disabled, kubectl will need to use an admin-authenticated kubeconfig.
   local master_ip=${1:-localhost}
   if [[ -n "${KUBE_BOOTSTRAP_TOKEN:-}" ]]; then
-    if [[ "${ENABLE_APISERVER_INSECURE_PORT:-false}" == "true" ]]; then
-      if [[ "${KUBERNETES_TENANT_PARTITION:-false}" == "true" ]]; then
-        create-kubeconfig "kube-bootstrap" "${KUBE_BOOTSTRAP_TOKEN}" "${PROXY_RESERVED_IP}" "8888" "http"
-      else
-        create-kubeconfig "kube-bootstrap" "${KUBE_BOOTSTRAP_TOKEN}" "${master_ip}" "8080" "http"
-      fi
+    if [[ "${KUBERNETES_TENANT_PARTITION:-false}" == "true" ]]; then
```

Comment:
this assumes if the RP or TP, it is always using the insecure port --- which is incorrect assumption. some hard coded TRUE for RP or TP in the config default have been removed. so we need to keep the ENABLE secure port check here.

Reply:
At this time the assumption is correct. When we add support for secure in scaleout, we will change this to use https by default.